roles/module_network_validated_content/files/health_checks/templates/eos_show_ip_bgp_summary.yaml

```diff
@@ -0,0 +1,50 @@
+# fmt: off
+[
```

Comment:
@ashwini-mhatre  can all the scaffolded files be of .yml type instead of .yaml
Regards

Reply:
Thanks for reviewing will make these changes